Dockerfile

```diff
@@ -0,0 +1,37 @@
+# base image (host os) -> python3.8.10
+FROM python:3.8.10
```

Comment:
@haniyenmati this might be too heavy for what we do
Can we switch to `alpine` version?